webhooks/external_account_config.go

```diff
@@ -0,0 +1,87 @@
+package webhooks
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+)
+
+/*
+
+Adapted from:
```

Comment:
nit: "Adopted" maybe?

Reply:
I think both makes sense. It was both adopted from, and adapted (because I had to add the `json` field attributes) to the struct fields.

Reply:
Understood. Thank you for the explanation!